client/src/pages/gratitude.tsx

```diff
@@ -51,7 +52,10 @@ function Page(props: Props) {
       form.resetFields();
       message.success('Your feedback has been submitted.');
     } catch (e) {
-      message.error('An error occurred. Please try later.');
+      const error = e as AxiosError<any>;
+      const response = error.response;
+      const errorMessage = response?.data?.message ?? 'An error occurred. Please try later.';
+      message.error(errorMessage);
```

Comment:
I also updated the error handler in gratitudes page to reflect the error sent in case of sending to yourself